apps/email/js/cards/compose.js

```diff
@@ -451,6 +467,37 @@ return [
       return bubble;
     },
 
+    getExistingEntriesForAutocomplete: function(inputNode) {
```

Comment:
Should this function just return the union of all the recipients lists instead of scoping by the list?  This implementation seems to assume that I'd want to, for example, include you on both the to and the cc lists, but I don't see why I'd want to do that.  (And both bcc-ing someone and including them on a non-bcc list also seems weird.)